OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -32,10 +32,10 @@ public class HarvesterInfo : ITraitInfo, Requires<MobileInfo>
 		[Desc("How much resources it can carry.")]
 		public readonly int Capacity = 28;
 
-		public readonly int LoadTicksPerBale = 4;
+		public readonly int BaleLoadDelay = 4;
 
 		[Desc("How fast it can dump it's carryage.")]
```

Comment:
If everything is ticks we could add it to the header of the trait documentation like Westwood did it:
```ini
; RULE*.INI
; *** Red Alert 2 Rules ***
; If placed in game directory, it will override built in values. Values to be used as multipliers
; or percentages can be specified as either a simple floating point number (embed ".") or as a
; conventional percentage number (append "%"). Values used as distances or time delays
; are specified as simple floating point number. Distance values are expressed in cells. Time
; values are expressed in minutes.
```